src/scripts/app.js

```diff
@@ -1100,6 +1101,8 @@ export default class InteractiveBook extends H5P.EventDispatcher {
       config.behaviour.displaySummary === undefined ||
       config.behaviour.displaySummary;
 
+    config.behaviour.displaySubmitButton = config.behaviour.displaySubmitButton === undefined ? false : config.behaviour.displaySubmitButton;
```

Comment:
It's so much more idiomatic to let undefined elements remain undefined.

if (undefined) { ... } is just as efficient and normal as if (false) { ... }, and both evaluate equivalently.  Just delete this line